compute/admin_guide/configure/configure-agentless-scanning.adoc

```diff
@@ -12,7 +12,31 @@ ifdef::compute_edition[]
 
 * Ensure you have permissions to create service keys and security groups in your cloud account.
 * Ensure you have permissions to apply agentless permission templates to your cloud account.
-* Ensure you can connect to the Prisma Cloud Console over HTTPS from your cloud account. If default security group is not available, create custom security group with custom VPC that allows connection for scanners from the account to Prisma Cloud Console.  
+* Ensure you can connect to the Prisma Cloud Console over HTTPS from your cloud account. If default security group is not available, create custom security group with custom VPC that allows connection for scanners from the account to Prisma Cloud Console. A recommended security group should deny all incoming traffic and allow one egress port only which is tcp/443 for the SaaS and tcp/8083 for the self-hosted console.
+
+----
```

Comment:
Thank you for clarifying what the recommended security group should do, but please do not include the JSON configuration here. 
It would be confusing for the reader since this is not part of the flow they need to complete to configure agentless scanning but rather a prerequisite.